### DISCUSSION TITLE
Configuring a "secondary" button in custom theme

### DISCUSSION THREAD

**Discussion by liquidz00** [TomSchimansky/CustomTkinter#1661] [Q&A]
Apologies if this has been asked and answered before, I searched but could not find any relevant information on this topic. 

I am creating my own color theme for my project. Was hoping to have a "primary" button style and a "secondary" button style to use for my CTkButton widgets. I am not looking for the buttons to function any differently, just adjust their styling. 

Is this possible? Thank you for any help in advance!

**Comment by avalon60** [comment]
Within a given theme, the styling of buttons has singular attributes (shape, fonts), except for the colour of buttons.  The latter have Dark and Light options. So you can make  them change colour between Light and Dark modes. 

You can of course override the theme settings programatically.
